src/main/java/socialite/ui/PersonCard.java

```diff
@@ -282,6 +283,10 @@ private void handleButtonAction() {
         content.putString(person.toSharingString());
         clipboard.setContent(content);
 
+        // Show the copied info in result display
+        ResultDisplay resultDisplay = MainWindow.getWindow().resultDisplay;
+        resultDisplay.setFeedbackToUser(String.format(ShareCommand.MESSAGE_SHARE_PERSON_SUCCESS, content));
```

Comment:
Seems like you need to set it here as the share button logic is completely handled within the UI, would it be better (or possible) for the button to "trigger" a command? But this would cause a few problems, such as having an extra `share` command in the commandHistory. 

Another suggestion I can think of would be to have a `setFeedbackToUser` method in the mainWindow that just calls the `setFeedbackToUser` method of the resultDisplay, similar to the methods in the `ModelManager` or `StorageManager` facade classes. Since mainWindow is already a singleton class, this would allow the resultDisplay to remain private.

One last suggestion would be to make the `ResultDisplay` class a singleton class too, so that you can get the resultDisplay directly instead of through the MainWindow.

 Let me know what you think!

Reply:
Thanks for the comment Zac! I think second option would be the best one of the three, will be amending to that solution in a separate commit!